packages/http-client-csharp/emitter/src/type/input-examples.ts

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+import { InputParameter } from "./input-parameter.js";
+import {
+  InputArrayType,
+  InputDictionaryType,
+  InputLiteralType,
+  InputModelType,
+  InputNullableType,
+  InputPrimitiveType,
+  InputType,
+  InputUnionType,
+} from "./input-type.js";
+import { OperationResponse } from "./operation-response.js";
+
+interface InputExampleBase {
+  kind: string;
+  name: string;
+  description: string;
+  filePath: string;
+  rawExample: any;
+}
+
+export interface InputHttpOperationExample extends InputExampleBase {
+  kind: "http";
+  parameters: InputParameterExample[];
+  responses: Map<number, OperationResponseExample>;
+}
+
+export interface InputParameterExample {
+  parameter: InputParameter;
+  value: InputTypeExample;
+}
+
+export interface OperationResponseExample {
+  response: OperationResponse;
+  // headers: SdkHttpResponseHeaderExample[];
+  bodyValue?: InputTypeExample;
+}
+
+export type InputTypeExample =
```

Comment:
Just curious. We really have examples for primitive types?

Reply:
Of course.
In the example of an operation, we need values for all its parameters, and a parameter may be a string, and we need an example for this primitive types.